templates/artist/includes/artist_detail_overview.html

```diff
@@ -21,6 +21,7 @@
                 <h4>AMOUNT RAISED</h4>
                 <h4 class="investing">${{ campaign.amount_raised|intcomma }}</h4>
                 <h5>OUT OF ${{ campaign.amount|intcomma }}</h5>
+                <p class="days-remaining">WITH {{ campaign.days_remaining }} DAYS REMAINING</p>
```

Comment:
Change to:
```html
<p class="days-remaining">WITH {{ campaign.days_remaining }} DAY{{ campaign.days_remaining|pluralize|upper }} REMAINING</p>
```
so that it shows as WITH 1 DAY REMAINING at 1 day (instead of 1 DAYS REMAINING).